src/Compilers/CSharp/Test/Symbol/Compilation/CompilationAPITests.cs

```diff
@@ -2053,6 +2053,30 @@ public void PreviousSubmissionWithError()
             Assert.Throws<InvalidOperationException>(() => CreateSubmission("a + 1", previous: s0));
         }
 
+        [Fact]
+        public void CreateTupleTypeSymbol()
```

Comment:
I think it would be inappropriate to annotate the class because it is not specific to tuples. Perhaps the individual tests should be annotated instead. I personally would keep all tuple specific test in one file rather than spreading them around.